client/my-sites/importer/newsletter/summary.tsx

```diff
@@ -97,11 +97,14 @@ export default function Summary( {
 							),
 						}
 					) }
-					<img
-						src={ pauseSubstackBillingImg }
-						alt={ __( 'Pause Substack billing' ) }
-						className="pause-billing"
-					/>
+					<details>
```

Comment:
I was hoping there to be a Reach component somewhere with ready made styling.

I remember someone took [FoldableCard](https://github.com/Automattic/wp-calypso/tree/637b2e22a34f6bf5c5534ac8f489c689f1495c1e/packages/components/src/foldable-card) and re-styled it to look much simpler like this, so that could be somewhere around Calypso.